manifests/init.pp

```diff
@@ -18,6 +18,12 @@
 #   Should this module manage the Unix system group under which BIND runs (see
 #   dns::params)?  Defaults to true. Set to false if you want to manage the
 #   system group yourself.
+# @param manage_service
+#   Should this module manage the dns service?
+#   This only applies to the service management (running, stopped) and not to
+#   whether the service should be installed or not.
+#   Calls to "notify => Service['$dns::namedservicename']" will still be
```

Comment:
That's actually going to be a problem I think. Please add tests for that. I suspect we need to change those to `Class['dns::service']` instead.

Reply:
In my mind, the calls should always happen because otherwise configuration changes will not be picked up by the service until a manual restart is issued.
Do you think we should inhibit them instead (I'm 99% sure that's what calling `Class['dns::service']` would do, but please correct me if I'm wrong)?

Also, tests. You're right. Forgot about them.
I need to learn Ruby unit testing, might take me a little bit.

Reply:
Correct. If you notify `Class['dns::service']` it'd work. It respects the `manage_service` parameter and knows the correct service name.

If you need help with the tests, let me know. I'm planning a release next week so now's the ideal time to get your changes in :)